src/cryptojwt/key_bundle.py

```diff
@@ -1246,3 +1280,19 @@ def init_key(filename, type, kid="", **kwargs):
     _new_key = key_gen(type, kid=kid, **kwargs)
     dump_jwk(filename, _new_key)
     return _new_key
+
+
+def key_by_alg(alg: str):
+    if alg.startswith("RS"):
+        return key_gen("RSA", alg="RS256")
```

Comment:
`alg` is not used by `key_gen` I believe, remove?

Reply:
Actually it is possible to use.
For the case where you want to limit the usage of for instance an RSA key to only one algorithm.